src/store/traction/pacbio/runCreate/state.js

```diff
@@ -12,11 +12,31 @@ export default () => {
       // The SMRT Link version store.
       smrtLinkVersions: {},
     },
+    // Run: The current run being edited or created
+    run: {},
 
+    // Wells: The wells that belong to the run
+    wells: {},
+
+    //Pools: The pools that belong to the wells or the pool selected for a new run
     pools: {},
```

Comment:
Just wondering when tubes/libraries/requests/tags are all used. Are they just used to display information in the pool list? If thats the case i am wondering if its worth splitting them all out into separate store objects if they just need to be rejoined anyway. Maybe store all of it nested within the pools? Just a thought, may be a reason we cant do it.

Reply:
I would suggest we tackle this as part of the Revio work ...